### PR TITLE
[TEVA-3520] New homepage

### PR DESCRIPTION
**Pull request by cesidio**
## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-3520

## Screenshots of UI changes:
Screenshot from 2022-01-11 12-23-41

### CODE REVIEW

File: app/frontend/src/styles/application/home.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
+.app-search-form-container {
+  padding: govuk-spacing(4);
+  color: #fff;
+  background: url('../../images/TVS-logo.png') no-repeat 100% 10px/120px auto govuk-colour("dark-blue");
```
Comment (elliotcm) [inline app/frontend/src/styles/application/home.scss:4]: Does this play nicely with the asset pipeline?

Comment (cesidio) [reply app/frontend/src/styles/application/home.scss]: Yes, it renders the image as you can see in the review app.

File: config/locales/en.yml [hunk 2]
```diff
@@ -237,13 +245,10 @@ en:
...
             sign_up: sign up
+        title: Advertise a school job
       title: Find a job in teaching
-    signed_in_jobseeker:
-      description: Apply for jobs you are interested in, and check the status of your applications.
-      title: Apply for jobs
-    signed_out_jobseeker:
-      description_html: "%{link} or create an account to apply for jobs that use the Teaching Vacancies application form."
-      title: Apply for jobs
+      title_h1: Find teacher and school jobs near you
```
Comment (josephhull676) [inline config/locales/en.yml:250]: Could `title` and `subtitle` be better?

Comment (cesidio) [reply config/locales/en.yml:250]: Maybe. I like that these title are grouped together and are easy to find starting all with title, and the HTML tag they are pertinent to

File: app/frontend/src/styles/application/home.scss [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  .govuk-button {
+    width: auto;
+  }
+}
+
+.app-search-form-inner {
+  @include govuk-media-query($from: tablet) {
+    align-items: end;
+    display: flex;
+
```
Comment (josephhull676) [inline app/frontend/src/styles/application/home.scss:46]: gap: 1%;

File: app/frontend/src/styles/application/home.scss [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    > div {
+      padding-right: 30px;
+      width: 33.33%;
```
Comment (josephhull676) [inline app/frontend/src/styles/application/home.scss:49]: `flex-basis: 33%`

Comment (josephhull676) [reply app/frontend/src/styles/application/home.scss:49]: With this and gap above, you can get rid of the padding I think

**Review by josephhull676 — APPROVED** [review]

**Pull request merged**